images/scripts/rhel4edge.bootstrap

```diff
@@ -55,8 +55,33 @@ if [ "$STATUS" != "SUCCESS" ]; then
     echo "Failed to build rhel4edge-$timestamp"
     exit 1
 fi
+ISO_FILE="rhel4edge-$timestamp.iso"
+curl --netrc-file /secrets/crc_passwd -L "https://console.redhat.com/api/edge/v1/storage/isos/$ID" -o "$ISO_FILE"
+
+# ==========================================
+# customize fleet.ks file inside ISO, to be able to do installation without user interaction
+DIR_NAME=$(mktemp -d -p $BASE)
+FLEET_FILE="$DIR_NAME/fleet.ks"
+
+mv "$ISO_FILE" "$DIR_NAME/fleet_source.iso"
+chmod 775 "$DIR_NAME"
+# extract fleet.ks to directory
+xorriso -osirrox on -indev "$DIR_NAME/fleet_source.iso" -extract /fleet.ks "$DIR_NAME/fleet.ks"
+# this is important to do for automatic installation
+sed -i "/network --boot/a rootpw --lock" "$DIR_NAME/fleet.ks"
+
+curl -L "https://raw.githubusercontent.com/loadtheaccumulator/fleet-iso-util/main/fleetkick.sh" -o "$DIR_NAME/fleetkick.sh"
+chmod a+x "$DIR_NAME/fleetkick.sh"
+# change workdir in script (it is used inside podman, so it expects specific path)
+sed -i "s@BASEDIR=.*@BASEDIR='$DIR_NAME'@" "$DIR_NAME/fleetkick.sh"
+mkdir "$DIR_NAME/workdir"
+
+$DIR_NAME/fleetkick.sh -w "$DIR_NAME/workdir"
```

Comment:
Just about everything about this is wrong: Conceptually we are not testing the built rhel4edge image, but a heavily hacked rebuild of it. Operationally, this is untrusted arbitrary code execution inside the RH network with access to secrets.

Customizations should happen either via a kickstart file (which then runs *inside* the built VM), which can be served over local http instead of using the existing `fleet.ks` on the image; or as virt-install arguments in images/scripts/virt-install-rhel4edge; or perhaps edge-request.json has some customization options here?